File: server/app/models/enums/runway_surface_types.py
from enum import Enum


class RunwaySurfaceTypes(str, Enum):
    ASPHALT = "Asphalt"
    CONCRETE = "Concrete"
    GRASS = "Grass"
    GRAVEL = "Gravel"
    DIRT = "Dirt"
    COMP_MATERIAL = "Composite Materials"

File: server/src/main/java/com/flightcoordinator/server/enums/RunwaySurfaceTypes.java
package com.flightcoordinator.server.enums;

// TODO turn into an endpoint

public enum RunwaySurfaceTypes {
  ASPHALT("Asphalt"),
  CONCRETE("Concrete"),
  GRASS("Grass"),
  GRAVEL("Gravel"),
  DIRT("Dirt"),
  COMP_MATERIAL("Composite Materials");

  public final String type;

  private RunwaySurfaceTypes(String type) {
    this.type = type;
  }
}
